Corgibytes.Freshli.Cli.Test/CommandOptions/ScanCommandOptionsTest.cs

```diff
@@ -12,53 +12,53 @@
 using Xunit;
 using Xunit.Abstractions;
 
-namespace Corgibytes.Freshli.Cli.Test.CommandOptions
+namespace Corgibytes.Freshli.Cli.Test.CommandOptions;
+
+public class ScanCommandOptionsTest : FreshliTest
```

Comment:
Haha nice! See my Daily Journal: https://github.com/corgibytes/the-oracle/wiki/Edwin's-Daily-Journal#tuesday-june-14-2022
For some reason Rider did not want to follow my file template for this. So I'm glad we are switching to this.